code/modules/mob/living/emote.dm

```diff
@@ -230,6 +230,16 @@
 	message = "jumps!"
 	hands_use_check = TRUE
 
+/datum/emote/living/jump/run_emote(mob/living/user, params, type_override, intentional)
+	. = ..()
+	if(!.)
+		return
```

Comment:
```suggestion
		return FALSE
```

this is just so that we remain nice to consumers of this proc, since parent returns `FALSE`.